plugins/base/frontend/src/main/components/search/search.tsx

```diff
@@ -43,15 +44,30 @@ const WithFuzzySearchFilterComponent: React.FC<Props> = ({data}: Props) => {
     )
 }
 
+const templateGenerator = (page:Page) => {
+    let classGenerator = (page:Page) => {
+        let classes = ""
+        if(page.level !== undefined) classes = classes + " indented"
+        if(page.disabled) classes = classes + "disabled"
```

Comment:
no space before `disabled`? This can lead to class like ` intenteddisabled`

Reply:
Added space before disabled